[DEVDOCS-5816]: [update] Apps Publishing, update screenshots and first step details (#191)
bigcommerce/docs: docs/api-docs/apps/guide/apps-13-publishing.mdx

* App summary
* App category

## Provide technical information

> To begin the app certification process, provide the technical details of the app. Specify app type, multiple users support, callback URLs, and OAuth scopes; and, provide detailed testing instructions.

Reviewer: ```suggestion
Provide the app's technical details to begin the app certification process. Specify the app type, multiple user support, callback URLs, and OAuth scopes, and provide detailed testing instructions.
```

![Technical](https://storage.googleapis.com/bigcommerce-production-dev-center/Big%20Design%20Docs/Technical%20-Callback%20URLS.png "Technical")

| Field                  | Description                                                                |
|:-----------------------|:---------------------------------------------------------------------------|

> | Multiple Users         | Optionally allow your app to be accessible to store users other than owner |

Reviewer: ```suggestion
| Multiple Users         | Optionally allow your app to be accessible to store users other than the owner |
```

> | App Summary     | App tagline used in the marketplace category and search results; 128 character max                     |

Reviewer: ```suggestion
| App Summary     | App tagline used in the marketplace category and search results; 128 characters max                     |
```
